folium/plugins/mouse_position.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+# CONVERTED TO PYTHON FROM: https://github.com/ardhi/Leaflet.MousePosition
```

Comment:
You are using the plugin and not really converting it to Python.
I'm not a lawyer but I don't think this is necessary.